File: SDKs/ote-godaddy.com/certificates/1.0.0/typescript/src/sdk/models/operations/certificatealternateemailaddress.ts
import { SpeakeasyMetadata, SpeakeasyBase } from "../../../internal/utils";



export class CertificateAlternateEmailAddressPathParams extends SpeakeasyBase {
  @SpeakeasyMetadata({ data: "pathParam, style=simple;explode=false;name=certificateId" })
  certificateId: string;

  @SpeakeasyMetadata({ data: "pathParam, style=simple;explode=false;name=emailAddress" })
  emailAddress: string;
}


export class CertificateAlternateEmailAddressRequest extends SpeakeasyBase {
  @SpeakeasyMetadata()
  pathParams: CertificateAlternateEmailAddressPathParams;
}


export class CertificateAlternateEmailAddressResponse extends SpeakeasyBase {
  @SpeakeasyMetadata()
  certificateEmailHistory?: any;

  @SpeakeasyMetadata()
  contentType: string;

  @SpeakeasyMetadata()
  error?: any;

  @SpeakeasyMetadata()
  statusCode: number;
}

File: SDKs/ote-godaddy.com/certificates/1.0.0/python/src/sdk/models/operations/certificate_alternate_email_address.py
import dataclasses
from typing import Any,Optional


@dataclasses.dataclass
class CertificateAlternateEmailAddressPathParams:
    certificate_id: str = dataclasses.field(metadata={'path_param': { 'field_name': 'certificateId', 'style': 'simple', 'explode': False }})
    email_address: str = dataclasses.field(metadata={'path_param': { 'field_name': 'emailAddress', 'style': 'simple', 'explode': False }})
    

@dataclasses.dataclass
class CertificateAlternateEmailAddressRequest:
    path_params: CertificateAlternateEmailAddressPathParams = dataclasses.field()
    

@dataclasses.dataclass
class CertificateAlternateEmailAddressResponse:
    content_type: str = dataclasses.field()
    status_code: int = dataclasses.field()
    certificate_email_history: Optional[Any] = dataclasses.field(default=None)
    error: Optional[Any] = dataclasses.field(default=None)
    
